src/core/src/bootstrap/Constants.py

```diff
@@ -62,20 +62,32 @@ class ConfigSettings(EnumBackport):
         PATCHES_TO_INCLUDE = 'patchesToInclude'
         PATCHES_TO_EXCLUDE = 'patchesToExclude'
         MAINTENANCE_RUN_ID = 'maintenanceRunId'
+        PATCH_MODE = 'patchMode'
 
-    # File to save default settings for auto OS updates
+        # File to save default settings for auto OS updates
     IMAGE_DEFAULT_PATCH_CONFIGURATION_BACKUP_PATH = "ImageDefaultPatchConfiguration.bak"
 
     # Operations
     ASSESSMENT = "Assessment"
     INSTALLATION = "Installation"
+    CONFIGURE_PATCHING = "ConfigurePatching"
     PATCH_ASSESSMENT_SUMMARY = "PatchAssessmentSummary"
     PATCH_INSTALLATION_SUMMARY = "PatchInstallationSummary"
     PATCH_METADATA_FOR_HEALTHSTORE = "PatchMetadataForHealthStore"
+    CONFIGURE_PATCHING_SUMMARY = "ConfigurePatchingSummary"
 
     # patch versions for healthstore when there is no maintenance run id
     PATCH_VERSION_UNKNOWN = "UNKNOWN"
 
+    # Patch Modes for Configure Patching
+    DEFAULT_FOR_IMAGE = "DefaultForImage"  # todo: cross check with CRP
```

Comment:
It's ImageDefault --- is DefaultForImage mentioned anywhere?? Please share if you've seen it like this somewhere..

Reply:
Updated. This was likely done when we were still using DefaultForImage in CRP.